homeassistant/components/switch/amcrest.py

```diff
@@ -1,6 +1,5 @@
 """
 Support for toggling Amcrest IP camera settings.
-
```

Comment:
I think it's this space, most integrations have a new line between the `Support for ...` line and `For more details ...`